### PR TITLE
Update for the Bureau Structure

### PR DESCRIPTION
**Pull request by sebworks**
Update for Bureau Structure

## Changes

- Updated theBureau Structure.

## Testing

- Visit `http://localhost:8000/about-us/the-bureau/bureau-structure/` and verify the correct structure.

## Review

- @willbarton 
- @virginiacc 
- @ajbush 

## Screenshots
screen shot 2016-08-03 at 5 49 02 pm
screen shot 2016-08-03 at 5 48 35 pm
screen shot 2016-08-03 at 5 48 24 pm
screen shot 2016-08-03 at 5 48 04 pm
screen shot 2016-08-03 at 5 47 48 pm

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by sebworks** [comment]
@ajbush, do we need a new png file?

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html [hunk 1]
```diff
@@ -201,7 +201,7 @@
     }, {
         'titles': ['Deputy General Counsel', 'Assistant Director'],
         'slug':   'litigation-and-oversight',
-        'name':   'Vacant',
+        'name':   '(continuing) Vacant',
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: Does `continuing` need capitalization?

Comment (sebworks) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: @sarahsimpson09, can you provide guidance?

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html [hunk 2]
```diff
@@ -186,12 +186,12 @@
     'children': [{
         'titles': ['Deputy Associate Director'],
         'slug':   'principal-deputy-general-counsel',
-        'name':   'Vacant',
+        'name':   'Rich Lepley',
```
Comment (virginiacc) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: This is very minor, but the png version uses 'Richard Lepley' here. Should this be the same, or are both fine?

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html (hunk 1 above)
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: "(continuing)" was just for us -- to let us know that the position is *still* vacant even with these updates. It should not be published.

Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: @ajbush we'll need to make that update to the PNG as well

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html (hunk 2 above)
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: I will double check this, thanks @virginiacc

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html [hunk 3]
```diff
@@ -20,9 +20,9 @@
         'name':   'Jeffrey Sumberg',
         'title':  'Human Capital'
         }, {
-        'titles': ['Chief Information Officer (CIO)', 'Assistant Director'],
+        'titles': ['Chief Information Officer (CIO)', 'Assistant Director<sup>*</sup>'],
         'slug':   'technology-and-innovation',
```
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: This title should be "Acting Chief Information Officer (CIO)"

Comment (sebworks) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: Should we add "Acting" to the rest of the positions with the same status? We typically just rely upon the symbol / legend.

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html (hunk 1 above)
Comment (sebworks) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/6c523ee9eea789273e70555bea46bdb6657e4d8a

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html (hunk 3 above)
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: You are correct @sebworks -- my apologies! Onward with usage of the asterisk!

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html (hunk 2 above)
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: Please use the full name, "Richard Lepley"

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html [hunk 4]
```diff
@@ -46,7 +46,7 @@
 
 {% block content_main %}
     <h1>Bureau Structure</h1>
-    <p>Org chart last updated: May 3, 2016</p>
+    <p>Org chart last updated: August 3, 2016</p>
```
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html]: I double checked with the stakeholder -- this date should be July 26, 2016. @ajbush will make that change in the png and upload it here for you.

Comment (sebworks) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/47806847c47c1c2cf6b344c23e4223df2fe355c5

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html (hunk 3 above)
Comment (sebworks) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/e5717da239d2ddf3eecb20eece06cfb3888553ca

File: cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html [hunk 5]
```diff
@@ -46,7 +46,7 @@
 
 {% block content_main %}
     <h1>Bureau Structure</h1>
-    <p>Org chart last updated: August 3, 2016</p>
+    <p>Org chart last updated: July 26, 2016</p>
     <div class="org-chart">
```
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html]: @sebworks thanks. Looks the png still needs to be updated though -- AJ uploaded one that has the July 26th update date on it.

Comment (sebworks) [inline cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html]: ok, updated in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/1d811548ae8483d37b9a630958b59e559c6d5aa4

**Comment by schaferjh** [comment]
Looks good, thanks! 👍

**Pull request merged**